src/features/visualization/d3-viz/setup-tooltip.js

```diff
@@ -1,5 +1,11 @@
-import { map, join, path } from "ramda";
-import { select, event as d3Event } from "d3-selection";
+/*
```

Comment:
If the intent is to remove the reliance on this tooltip then actually delete this file and remove any references instead of leaving dead/non-executing code in place.